images/win/scripts/Installers/Validate-PostgreSQL.ps1

```diff
@@ -0,0 +1,32 @@
+function Validate-PostgreSQL {
+    $pgready = Start-Process -FilePath pg_isready -Wait -PassThru
+    $exitCode = $pgready.ExitCode
+    if ($exitCode -eq 0)
+    {
+        Write-Host -Object "PostgreSQL has been successfully installed."
+    }
+    else
+    {
+        Write-Host -Object "PostgreSQL is not ready. Exitcode: $exitCode"
+        exit $exitCode
+    }
+}
+
+$paths=(Get-CimInstance Win32_Service -Filter "Name LIKE 'postgresql-%'").PathName
```

Comment:
Could you please fix styling for both scripts?
Use spaces before and after `=` like `$pgroot = $paths.split...`

Reply:
Thank you, I've changed.